src/components/fileUpload/index.js

```diff
@@ -5,104 +5,21 @@ import React from "react";
 import glamorous from "glamorous";
 import PropTypes from "prop-types";
 
-import * as COLOUR from "govuk-colours";
-import { BREAKPOINTS, SITE_WIDTH } from "../../constants/index";
+import { STYLE_DEFAULT } from "../../constants/index";
 
-const mediaQueries = {
-  largeScreen: `@media only screen and (min-width: ${BREAKPOINTS.LARGESCREEN})`
-};
-
-const Label = glamorous.label(
-  {
-    marginBottom: "15px",
-    display: "flex",
-    flexDirection: "column",
-    [mediaQueries.largeScreen]: {
-      maxWidth: SITE_WIDTH,
-      marginBottom: "30px"
-    }
-  },
-  ({ errorText }) => ({
-    borderLeft: errorText ? `4px solid ${COLOUR.ERROR_COLOUR}` : "",
-    marginRight: errorText ? `15px` : "0",
-    paddingLeft: errorText ? `10px` : "0"
-  })
+const Label = glamorous.label(STYLE_DEFAULT.LABEL, ({ errorText }) =>
+  STYLE_DEFAULT.LABEL.ERROR(errorText)
 );
 
-const LabelText = glamorous.span(
-  {
-    fontFamily: '"nta", Arial, sans-serif',
-    WebkitFontSmoothing: "antialiased",
-    MozOsxFontSmoothing: "grayscale",
-    fontWeight: 400,
-    fontSize: "16px",
-    lineHeight: "1.25",
-    color: `${COLOUR.BLACK}`,
-    display: "block",
-    paddingBottom: "2px",
-    [mediaQueries.largeScreen]: {
-      fontSize: "19px",
-      lineHeight: "1.31579"
-    }
-  },
-  ({ errorText }) => ({
-    fontWeight: errorText ? 700 : LabelText.fontWeight
-  })
+const LabelText = glamorous.span(STYLE_DEFAULT.LABEL_TEXT, ({ errorText }) =>
```

Comment:
this feels as though LabelText should be its own component in its own file rather than reconstructed with the same value in multiple places